doc/ch-image.rst

```diff
@@ -627,11 +638,29 @@ This one is (ironically) :code:`apt-get` failing to drop privileges:
   E: seteuid 100 failed - seteuid (22: Invalid argument)
   E: setgroups 0 failed - setgroups (1: Operation not permitted)
 
-By default, nothing is done to avoid these problems, though :code:`ch-image`
-does try to detect if the workarounds could help. :code:`--force` activates
-the workarounds: :code:`ch-image` injects extra commands to intercept these
-system calls and fake a successful result, using :code:`fakeroot(1)`. There
-are three basic steps:
+Charliecloud provides two different workaround to avoid these problems. Both
```

Comment:
"`workaround`" should be "`workarounds`" (plural) if you want to use that word, but I might suggest a synonym since you already use "workaround" frequently throughout this new text.  Maybe "`mechanisms`" or "`techniques`?"